### PR TITLE
[No QA] Simplify StagingDeployCash comparison URL

### PR DESCRIPTION
**Pull request by roryabraham**
### Details
I feel like a big dum-dum for not seeing this simplification earlier! Silver lining though: the contributor who worked on the `generateVersionComparisonURL` did set up a good baseline for unit testing the `GithubUtils`

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/158542

### Tests
1. Make sure the `StagingDeployCash` is not locked.
1. Merge this PR
1. The `StagingDeployCash` should be updated and the "compare changes" link should be this: https://github.com/Expensify/Expensify.cash/compare/production...staging

### Platforms
Github only, no changes in-product.

### CODE REVIEW

File: .github/libs/GithubUtils.js [hunk 1]
```diff
@@ -358,10 +279,8 @@ class GithubUtils {
                 }
 
                 issueBody += '\r\ncc @Expensify/applauseleads\r\n';
-
                 return issueBody;
             })
-            // eslint-disable-next-line no-console
             .catch(err => console.warn('Error generating comparison URL, continuing...', err));
```
Comment (Jag96) [inline .github/libs/GithubUtils.js:284]: Is this warning still accurate? It sounds like this should be something like `Error generating issue body`

**Review by Jag96 — DISMISSED** [review]
Nice improvement! Just one comment on a warning but LGTM

**Comment by deetergp** [comment]
> Merge this PR

I think you might be missing a link in your test steps.

**Review by deetergp — DISMISSED** [review]
I agree with Joe regarding the error message, but other than that, it looks good.

**Comment by roryabraham** [comment]
Updated the console warning 👍

**Review by Jag96 — APPROVED** [review]

**Review by deetergp — APPROVED** [review]

**Pull request merged**